.github/workflows/publish_image.yml

```diff
@@ -0,0 +1,78 @@
+name: publish_image
+
+on:
```

Comment:
PR description says:

Every main commit publishes an image to ECR with the commit hash
Terraform is redeployed with the latest hash

But what makes sure the Terraform is deployed after the ECR image is pushed?

Reply:
it has now been added